qemu/tests/balloon_check.py

```diff
@@ -37,6 +37,11 @@ def __init__(self, test, params, env):
             self.params["balloon_test_setup_ready"] = True
             self.ori_gmem = self.get_memory_status()
             self.current_gmem = self.ori_gmem
+            self.pre_mem = self.get_vm_mem(self.vm)
+            self.pre_gmem = self.ori_gmem
+        # ori_mem/gmem is original memory
+        # pre_mem/gmem is memory value before balloon test
+        # current_mem/gmem is current memory
```

Comment:
Looks like current_* and pre_* share the same meaning in code. Maybe we could remove one?

Reply:
They have different meaning, pre_* means memory before balloon, current_* means memory currently.

Reply:
But they are actually same, e.g. here, they have same values.

Reply:
Here is the very begging value set, do you see L347 L348, they are the value after balloon.

Reply:
It's not always the value after balloon, e.g. L205.
current_* can be totally replaced by pre_* in L205,  L308, L310 when it's in the right of '=', so I think it is useless.

Reply:
I still think keep self.pre_* and self.current_* is okay.
1. It's clear what they mean, memory value before and current.
2. If replace current_* with pre_*,  it's not understood well, why to set min_size with pre_*
 def get_memory_boundary(self, balloon_type=''):   
....
       if balloon_type == "enlarge":
            min_size = self.current_mmem
        elif balloon_type == "evict":
            max_size = self.current_mmem

Reply:
current_gmem is never been used, and current_mmem should always equal to get_ballooned_memory().
pre_* and current_* is not distinguished clearly. Before balloon, current_* is pre_*.  
If you think it's not appropriate to set min_size with pre_*, we can use get_ballooned_memory() here.

Reply:
Update. Delete current related value